marketplacetests/marketplace/regions/debug.py

```diff
@@ -21,6 +21,8 @@ def tap_back(self):
         self.marionette.find_element(*self._back_button_locator).tap()
 
     def select_region(self, region):
-        self.marionette.find_element(*self._region_select_locator).tap()
+        element = self.marionette.find_element(*self._region_select_locator)
+        self.marionette.execute_script('arguments[0].scrollIntoView(false);', [element])
```

Comment:
Could we add a comment mentioning the bug? If in the future we don't need to worry about v2.0 we should be able to remove this workaround. Something like `# Bug 937053 - tap() method should calculate elementInView from the coordinates of the tap`

With that change, r+